site/styles.css

```diff
@@ -193,17 +272,70 @@ input[type="submit"]:hover {
 
 .button-container {
     display: flex;
+    justify-content: center;
 }
 
 .button-container .button {
+    display: inline-block;
     margin-right: 16px;
 }
 
 .button-container .button:last-child {
     margin-right: 0;
 }
 
+.back-to-top {
```

Comment:
Could we add a border or something around the button? On mobile we can currently get issues like this: 

![deploy-preview-3--leedsjs netlify com_events_2019-05-29_(Galaxy S5)](https://user-images.githubusercontent.com/4780756/57585563-d3b79c00-74e1-11e9-97e3-f1062bda204b.png)

Reply:
I've made it a white circle with box-shadow instead... it should still contrast enough, see what you think :)